src/moments.jl

```diff
@@ -58,99 +84,184 @@ function var!(R::AbstractArray, A::RealArray, wv::WeightVec, dim::Int; mean=noth
+can be applied to the variance calculation if `corrected=true`.
+See [`var`](@ref) documentation for more details.
 """
-mean_and_var(A::RealArray) = (m = mean(A); (m, varm(A, m)))
+function mean_and_var(A::RealArray; corrected::Bool=true)
+    m = mean(A)
+    v = varm(A, m; corrected=corrected)
+    m, v
+end
 
 """
-    mean_and_std(x, [wv::WeightVec], [dim]) -> (mean, std)
+    mean_and_std(x, [wv::AbstractWeights], [dim]; corrected=false) -> (mean, std)
 
 Return the mean and standard deviation of a real-valued array `x`, optionally
 over a dimension `dim`, as a tuple. A weighting vector `wv` can be specified
-to weight the estimates. The weights are assumed to be frequency weights, also
-called case weights.
+to weight the estimates. Finally, bias correction can be applied to the
+standard deviation calculation if `corrected=true`.
+See [`std`](@ref) documentation for more details.
 """
-mean_and_std(A::RealArray) = (m = mean(A); (m, stdm(A, m)))
+function mean_and_std(A::RealArray; corrected::Bool=true)
+    m = mean(A)
+    s = stdm(A, m; corrected=corrected)
+    m, s
+end
 
-mean_and_var(A::RealArray, wv::WeightVec) = (m = mean(A, wv); (m, varm(A, wv, m)))
-mean_and_std(A::RealArray, wv::WeightVec) = (m = mean(A, wv); (m, stdm(A, wv, m)))
+function mean_and_var(A::RealArray, wv::AbstractWeights; corrected::DepBool=nothing)
+    m = mean(A, wv)
+    v = varm(A, wv, m; corrected=depcheck(:mean_and_var, corrected))
+    m, v
+end
+function mean_and_std(A::RealArray, wv::AbstractWeights; corrected::DepBool=nothing)
+    m = mean(A, wv)
+    s = stdm(A, wv, m; corrected=depcheck(:mean_and_std, corrected))
+    m, s
+end
 
-mean_and_var(A::RealArray, dim::Int) = (m = mean(A, dim); (m, varm(A, m, dim)))
-mean_and_std(A::RealArray, dim::Int) = (m = mean(A, dim); (m, stdm(A, m, dim)))
 
-mean_and_var(A::RealArray, wv::WeightVec, dim::Int) = (m = mean(A, wv, dim); (m, varm(A, wv, m, dim)))
-mean_and_std(A::RealArray, wv::WeightVec, dim::Int) = (m = mean(A, wv, dim); (m, stdm(A, wv, m, dim)))
+function mean_and_var(A::RealArray, dim::Int; corrected::Bool=true)
+    m = mean(A, dim)
+    v = varm(A, m, dim; corrected=corrected)
+    m, v
+end
+function mean_and_std(A::RealArray, dim::Int; corrected::Bool=true)
+    m = mean(A, dim)
+    s = stdm(A, m, dim; corrected=corrected)
+    m, s
+end
 
 
-##### General central moment
+function mean_and_var(A::RealArray, wv::AbstractWeights, dim::Int;
+                      corrected::DepBool=nothing)
+    m = mean(A, wv, dim)
+    v = varm(A, wv, m, dim; corrected=depcheck(:mean_and_var, corrected))
+    m, v
+end
+function mean_and_std(A::RealArray, wv::AbstractWeights, dim::Int;
+                      corrected::DepBool=nothing)
+    m = mean(A, wv, dim)
+    s = stdm(A, wv, m, dim; corrected=depcheck(:mean_and_std, corrected))
+    m, s
+end
+
 
-function _moment2(v::RealArray, m::Real)
+
+##### General central moment
+function _moment2(v::RealArray, m::Real; corrected=false)
```

Comment:
This is called by `varm(x, w::AbstractWeights, m, [dim]; corrected=false)` earlier in the file.